docs/requirements.txt

```diff
@@ -6,3 +6,4 @@ sphinx==1.8.5
 sphinx_rtd_theme
 nbsphinx==0.8.7
```

Comment:
Side note: We probably want to update the doc stack at some point and support newer sphinx versions etc. But that's orthogonal to this PR.

Reply:
Agreed!